src/modules/notification/adapters/smsService.adapter.ts

```diff
@@ -81,7 +81,7 @@ export class SmsAdapter implements NotificationServiceInterface {
             const twilio = require('twilio');
             const client = twilio(this.accountSid, this.authToken);
             const message = await client.messages.create({
-                from: '+12563056567',
+                from: '+15735704535',
```

Comment:
@poojakarma Why this hardcoded can you take it from config or env

Reply:
done